src/main/java/com/redhat/cloud/notifications/RestRecipient.java

```diff
@@ -1,11 +1,14 @@
 package com.redhat.cloud.notifications;
 
 import com.fasterxml.jackson.annotation.JsonProperty;
+import jakarta.validation.constraints.Pattern;
 
 import java.util.List;
 
 public class RestRecipient {
 
+    private static final String EMAIL_VALIDATION_REGEX = "^\\S+@\\S+\\.\\S+$";
```

Comment:
This should be enough for us, but this is a funny and interesting talk by Dylan Beattie on emails :grin: 

https://youtu.be/mrGfahzt-4Q?si=9kjkrGEmyFMQeM3C